internal/provider/resource_gitlab_project_access_token.go

```diff
@@ -197,7 +186,7 @@ func resourceGitlabProjectAccessTokenDelete(ctx context.Context, d *schema.Resou
 	}
 
 	log.Printf("[DEBUG] Delete gitlab ProjectAccessToken %s", d.Id())
-	_, err = client.ProjectAccessTokens.DeleteProjectAccessToken(project, projectAccessTokenID, gitlab.WithContext(ctx))
+	_, err = client.ProjectAccessTokens.RevokeProjectAccessToken(project, projectAccessTokenID, gitlab.WithContext(ctx))
```

Comment:
Same thing here ...

Reply:
This is just an upstream name change in go-gitlab. No functionality has changed as far as I can tell.

Reply:
Yeah this was me, sorry for the trouble! Wanted to make the naming more inline with how the rest of the methods is used (the use the naming used in the docs)...